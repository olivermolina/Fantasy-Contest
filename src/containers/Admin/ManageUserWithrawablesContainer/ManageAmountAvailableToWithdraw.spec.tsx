import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import ManageAmountAvailableToWithdrawContainer from './ManageAmountAvailableToWithdrawContainer';

jest.mock('~/utils/trpc', () => ({
  trpc: {
    user: {
      users: {
        useQuery: jest.fn().mockReturnValue({
          isLoading: true,
          data: [
            {
              id: 'user-id',
              username: 'jdoe',
              Wallets: [
                {
                  balance: 100,
                  cashBalance: 100,
                  bonusCredits: 0,
                  amountAvailableToWithdraw: 100,
                },
              ],
            },
          ],
        }),
      },
    },
    admin: {
      saveUserWallet: {
        useMutation: jest.fn().mockReturnValue({
          isLoading: false,
          mutate: jest.fn(),
        }),
      },
    },
  },
}));

describe('ManageAmountAvailableToWithdrawContainer', () => {
  it('should render correctly', () => {
    const { container } = render(<ManageAmountAvailableToWithdrawContainer />);
    expect(container).toMatchSnapshot();
  });
});
