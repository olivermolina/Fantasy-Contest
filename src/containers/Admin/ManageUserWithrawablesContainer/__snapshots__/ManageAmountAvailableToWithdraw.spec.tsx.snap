// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ManageAmountAvailableToWithdrawContainer should render correctly 1`] = `
<div>
  <div
    aria-hidden="true"
    class="MuiBackdrop-root css-kq809n-MuiBackdrop-root"
    data-testid="backdrop-loading"
    style="opacity: 0; visibility: hidden;"
  >
    <div
      class="MuiBox-root css-rmz4vk"
    >
      <span
        class="MuiCircularProgress-root MuiCircularProgress-indeterminate MuiCircularProgress-colorInherit css-62e83j-MuiCircularProgress-root"
        role="progressbar"
        style="width: 55px; height: 55px;"
      >
        <svg
          class="MuiCircularProgress-svg css-1idz92c-MuiCircularProgress-svg"
          viewBox="22 22 44 44"
        >
          <circle
            class="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate MuiCircularProgress-circleDisableShrink css-79nvmn-MuiCircularProgress-circle"
            cx="44"
            cy="44"
            fill="none"
            r="20.2"
            stroke-width="3.6"
          />
        </svg>
      </span>
      <div
        class="MuiBox-root css-1o8dslu"
      >
        <div
          class="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault css-wn51m9-MuiAvatar-root"
        >
          <svg
            height="36"
            preserveAspectRatio="xMidYMid meet"
            role="loading"
            version="1.0"
            viewBox="0 0 176.000000 192.000000"
            width="28"
            xmlns="http://www.w3.org/2000/svg"
          >
            <circle
              fill="#fff"
              id="top"
            />
            <g
              fill="#2463eb"
              stroke="#2463eb"
              transform="translate(0.000000,192.000000) scale(0.100000,-0.100000)"
            >
              <path
                d="M385 1891 c-40 -10 -66 -27 -90 -59 -18 -25 -20 -45 -23 -229 l-3 -203 60 0 61 0 0 173 c0 94 4 178 8 185 7 10 111 12 488 10 l479 -3 3 -187 2 -188 66 0 65 0 -3 204 c-3 224 -5 230 -72 280 -27 20 -39 21 -519 23 -271 1 -505 -2 -522 -6z"
              />
              <path
                d="M153 1309 c-24 -12 -52 -37 -65 -57 l-23 -37 0 -536 0 -535 25 -37 c46 -68 81 -77 289 -77 l182 0 23 53 24 52 -102 5 c-74 4 -109 10 -132 24 -73 44 -86 129 -39 245 l28 71 143 0 143 0 -15 -35 c-22 -52 -19 -56 33 -53 l47 3 28 69 c15 38 28 75 28 83 0 10 -22 13 -97 13 -79 0 -104 4 -130 20 -87 52 -89 127 -12 315 62 150 121 235 193 280 47 29 56 30 185 35 l136 5 16 35 c9 19 20 45 23 58 l7 22 -448 0 c-434 0 -449 -1 -490 -21z"
              />
              <path
                d="M1255 1275 c-14 -30 -25 -57 -25 -59 0 -3 49 -6 109 -8 105 -3 111 -4 140 -32 59 -57 63 -120 15 -241 l-26 -65 -140 0 -140 0 5 23 c3 12 9 32 12 45 7 21 4 22 -37 22 l-45 0 -29 -72 c-16 -40 -30 -76 -32 -80 -2 -5 36 -8 85 -8 119 -1 166 -21 195 -86 26 -60 22 -92 -31 -223 -60 -148 -87 -197 -139 -253 -75 -81 -105 -92 -258 -96 l-131 -4 -16 -46 c-10 -26 -17 -50 -17 -54 0 -5 184 -8 408 -8 403 0 408 0 452 23 24 12 53 35 64 50 21 28 21 41 24 563 3 597 5 578 -66 632 -34 26 -39 27 -194 30 l-158 3 -25 -56z"
              />
              <path
                d="M843 948 c-8 -13 -63 -140 -63 -146 0 -2 23 -2 51 0 l51 3 29 75 c16 41 29 76 29 78 0 9 -90 -1 -97 -10z"
              />
              <path
                d="M933 493 c-48 -110 -48 -103 1 -103 51 0 55 4 89 93 l25 67 -44 0 -45 0 -26 -57z"
              />
            </g>
          </svg>
        </div>
      </div>
    </div>
  </div>
  <div
    class="w-full h-[75vh]"
  >
    <div
      aria-colcount="7"
      aria-multiselectable="false"
      aria-rowcount="2"
      class="MuiDataGrid-root MuiDataGrid-root--densityStandard MuiDataGrid-withBorderColor css-1lymaxv-MuiDataGrid-root"
      role="grid"
    >
      <div
        class="MuiDataGrid-toolbarContainer css-128fb87-MuiDataGrid-toolbarContainer"
      >
        <button
          aria-expanded="false"
          aria-haspopup="menu"
          aria-label="Select columns"
          class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeSmall MuiButton-textSizeSmall MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeSmall MuiButton-textSizeSmall css-1knaqv7-MuiButtonBase-root-MuiButton-root"
          id=":r0:"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiButton-startIcon MuiButton-iconSizeSmall css-y6rp3m-MuiButton-startIcon"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
              data-testid="ColumnIconIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M6 5H3c-.55 0-1 .45-1 1v12c0 .55.45 1 1 1h3c.55 0 1-.45 1-1V6c0-.55-.45-1-1-1zm14 0h-3c-.55 0-1 .45-1 1v12c0 .55.45 1 1 1h3c.55 0 1-.45 1-1V6c0-.55-.45-1-1-1zm-7 0h-3c-.55 0-1 .45-1 1v12c0 .55.45 1 1 1h3c.55 0 1-.45 1-1V6c0-.55-.45-1-1-1z"
              />
            </svg>
          </span>
          Columns
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </button>
        <button
          aria-expanded="false"
          aria-haspopup="true"
          aria-label="Show filters"
          class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeSmall MuiButton-textSizeSmall MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeSmall MuiButton-textSizeSmall css-1knaqv7-MuiButtonBase-root-MuiButton-root"
          data-mui-internal-clone-element="true"
          id=":r2:"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiButton-startIcon MuiButton-iconSizeSmall css-y6rp3m-MuiButton-startIcon"
          >
            <span
              class="MuiBadge-root css-1c32n2y-MuiBadge-root"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                data-testid="FilterListIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M10 18h4v-2h-4v2zM3 6v2h18V6H3zm3 7h12v-2H6v2z"
                />
              </svg>
              <span
                class="MuiBadge-badge MuiBadge-standard MuiBadge-invisible MuiBadge-anchorOriginTopRight MuiBadge-anchorOriginTopRightRectangular MuiBadge-overlapRectangular MuiBadge-colorPrimary css-1pi4uwz-MuiBadge-badge"
              >
                0
              </span>
            </span>
          </span>
          Filters
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </button>
        <button
          aria-expanded="false"
          aria-haspopup="menu"
          aria-label="Density"
          class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeSmall MuiButton-textSizeSmall MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeSmall MuiButton-textSizeSmall css-1knaqv7-MuiButtonBase-root-MuiButton-root"
          id=":r5:"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiButton-startIcon MuiButton-iconSizeSmall css-y6rp3m-MuiButton-startIcon"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
              data-testid="TableRowsIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M21,8H3V4h18V8z M21,10H3v4h18V10z M21,16H3v4h18V16z"
              />
            </svg>
          </span>
          Density
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </button>
        <button
          aria-expanded="false"
          aria-haspopup="menu"
          aria-label="Export"
          class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeSmall MuiButton-textSizeSmall MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeSmall MuiButton-textSizeSmall css-1knaqv7-MuiButtonBase-root-MuiButton-root"
          id=":r7:"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiButton-startIcon MuiButton-iconSizeSmall css-y6rp3m-MuiButton-startIcon"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
              data-testid="SaveAltIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M19 12v7H5v-7H3v7c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2v-7h-2zm-6 .67l2.59-2.58L17 11.5l-5 5-5-5 1.41-1.41L11 12.67V3h2z"
              />
            </svg>
          </span>
          Export
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </button>
        <div
          class="MuiBox-root css-1rr4qq7"
        />
        <div
          aria-label="Search"
          class="MuiFormControl-root MuiTextField-root css-3be3ve-MuiFormControl-root-MuiTextField-root-MuiDataGrid-toolbarQuickFilter"
        >
          <div
            class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-colorPrimary MuiInputBase-formControl MuiInputBase-adornedStart MuiInputBase-adornedEnd css-v4u5dn-MuiInputBase-root-MuiInput-root"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall css-ptiqhd-MuiSvgIcon-root"
              data-testid="SearchIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
              />
            </svg>
            <input
              aria-invalid="false"
              class="MuiInputBase-input MuiInput-input MuiInputBase-inputTypeSearch MuiInputBase-inputAdornedStart MuiInputBase-inputAdornedEnd css-c63i49-MuiInputBase-input-MuiInput-input"
              id=":r9:"
              placeholder="Search…"
              type="search"
              value=""
            />
            <button
              aria-label="Clear"
              class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeSmall css-fp9tcu-MuiButtonBase-root-MuiIconButton-root"
              tabindex="0"
              type="button"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall css-ptiqhd-MuiSvgIcon-root"
                data-testid="CloseIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
                />
              </svg>
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
      </div>
      <div
        class="MuiDataGrid-main css-204u17-MuiDataGrid-main"
      >
        <div
          class="MuiDataGrid-columnHeaders MuiDataGrid-withBorderColor css-1iyq7zh-MuiDataGrid-columnHeaders"
          style="min-height: 56px; max-height: 56px; line-height: 56px;"
        >
          <div
            class="MuiDataGrid-columnHeadersInner MuiDataGrid-columnHeadersInner--scrollable css-gl260s-MuiDataGrid-columnHeadersInner"
            role="rowgroup"
            style="transform: translate3d(0px, 0px, 0px);"
          >
            <div
              aria-rowindex="1"
              class="css-yrdy0g-MuiDataGrid-columnHeaderRow"
              role="row"
            >
              <div
                aria-colindex="1"
                aria-label="ID"
                aria-sort="none"
                class="MuiDataGrid-columnHeader MuiDataGrid-columnHeader--sortable MuiDataGrid-withBorderColor"
                data-field="id"
                role="columnheader"
                style="height: 56px; width: 0px; min-width: 0; max-width: 0;"
                tabindex="0"
              >
                <div
                  class="MuiDataGrid-columnHeaderDraggableContainer"
                  draggable="false"
                >
                  <div
                    class="MuiDataGrid-columnHeaderTitleContainer"
                  >
                    <div
                      class="MuiDataGrid-columnHeaderTitleContainerContent"
                    >
                      <div
                        aria-label=""
                        class="MuiDataGrid-columnHeaderTitle css-t89xny-MuiDataGrid-columnHeaderTitle"
                        data-mui-internal-clone-element="true"
                      >
                        ID
                      </div>
                    </div>
                    <div
                      class="MuiDataGrid-iconButtonContainer css-ltf0zy-MuiDataGrid-iconButtonContainer"
                    >
                      <button
                        aria-label="Sort"
                        class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeSmall css-1pe4mpk-MuiButtonBase-root-MuiIconButton-root"
                        tabindex="-1"
                        title="Sort"
                        type="button"
                      >
                        <svg
                          aria-hidden="true"
                          class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall MuiDataGrid-sortIcon css-ptiqhd-MuiSvgIcon-root"
                          data-testid="ArrowUpwardIcon"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M4 12l1.41 1.41L11 7.83V20h2V7.83l5.58 5.59L20 12l-8-8-8 8z"
                          />
                        </svg>
                        <span
                          class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                        />
                      </button>
                    </div>
                  </div>
                  <div
                    class="MuiDataGrid-menuIcon"
                  >
                    <button
                      aria-expanded="false"
                      aria-haspopup="menu"
                      aria-label="Menu"
                      class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeSmall MuiDataGrid-menuIconButton css-1pe4mpk-MuiButtonBase-root-MuiIconButton-root"
                      data-mui-internal-clone-element="true"
                      id=":rd:"
                      tabindex="-1"
                      type="button"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall css-ptiqhd-MuiSvgIcon-root"
                        data-testid="TripleDotsVerticalIcon"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M12 8c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2zm0 2c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm0 6c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z"
                        />
                      </svg>
                      <span
                        class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                      />
                    </button>
                  </div>
                </div>
                <div
                  class="MuiDataGrid-columnSeparator MuiDataGrid-columnSeparator--sideRight"
                  style="min-height: 56px; opacity: 1;"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiDataGrid-iconSeparator css-i4bv87-MuiSvgIcon-root"
                    data-testid="SeparatorIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M11 19V5h2v14z"
                    />
                  </svg>
                </div>
              </div>
              <div
                aria-colindex="2"
                aria-label="Username"
                aria-sort="none"
                class="MuiDataGrid-columnHeader MuiDataGrid-columnHeader--sortable MuiDataGrid-withBorderColor"
                data-field="username"
                role="columnheader"
                style="height: 56px; width: 0px; min-width: 0; max-width: 0;"
                tabindex="-1"
              >
                <div
                  class="MuiDataGrid-columnHeaderDraggableContainer"
                  draggable="false"
                >
                  <div
                    class="MuiDataGrid-columnHeaderTitleContainer"
                  >
                    <div
                      class="MuiDataGrid-columnHeaderTitleContainerContent"
                    >
                      <div
                        aria-label=""
                        class="MuiDataGrid-columnHeaderTitle css-t89xny-MuiDataGrid-columnHeaderTitle"
                        data-mui-internal-clone-element="true"
                      >
                        Username
                      </div>
                    </div>
                    <div
                      class="MuiDataGrid-iconButtonContainer css-ltf0zy-MuiDataGrid-iconButtonContainer"
                    >
                      <button
                        aria-label="Sort"
                        class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeSmall css-1pe4mpk-MuiButtonBase-root-MuiIconButton-root"
                        tabindex="-1"
                        title="Sort"
                        type="button"
                      >
                        <svg
                          aria-hidden="true"
                          class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall MuiDataGrid-sortIcon css-ptiqhd-MuiSvgIcon-root"
                          data-testid="ArrowUpwardIcon"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M4 12l1.41 1.41L11 7.83V20h2V7.83l5.58 5.59L20 12l-8-8-8 8z"
                          />
                        </svg>
                        <span
                          class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                        />
                      </button>
                    </div>
                  </div>
                  <div
                    class="MuiDataGrid-menuIcon"
                  >
                    <button
                      aria-expanded="false"
                      aria-haspopup="menu"
                      aria-label="Menu"
                      class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeSmall MuiDataGrid-menuIconButton css-1pe4mpk-MuiButtonBase-root-MuiIconButton-root"
                      data-mui-internal-clone-element="true"
                      id=":rj:"
                      tabindex="-1"
                      type="button"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall css-ptiqhd-MuiSvgIcon-root"
                        data-testid="TripleDotsVerticalIcon"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M12 8c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2zm0 2c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm0 6c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z"
                        />
                      </svg>
                      <span
                        class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                      />
                    </button>
                  </div>
                </div>
                <div
                  class="MuiDataGrid-columnSeparator MuiDataGrid-columnSeparator--sideRight"
                  style="min-height: 56px; opacity: 1;"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiDataGrid-iconSeparator css-i4bv87-MuiSvgIcon-root"
                    data-testid="SeparatorIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M11 19V5h2v14z"
                    />
                  </svg>
                </div>
              </div>
              <div
                aria-colindex="3"
                aria-label="Cash Balance"
                aria-sort="none"
                class="MuiDataGrid-columnHeader MuiDataGrid-columnHeader--alignRight MuiDataGrid-columnHeader--sortable MuiDataGrid-columnHeader--numeric MuiDataGrid-withBorderColor"
                data-field="cashBalance"
                role="columnheader"
                style="height: 56px; width: 0px; min-width: 0; max-width: 0;"
                tabindex="-1"
              >
                <div
                  class="MuiDataGrid-columnHeaderDraggableContainer"
                  draggable="false"
                >
                  <div
                    class="MuiDataGrid-columnHeaderTitleContainer"
                  >
                    <div
                      class="MuiDataGrid-columnHeaderTitleContainerContent"
                    >
                      <div
                        aria-label=""
                        class="MuiDataGrid-columnHeaderTitle css-t89xny-MuiDataGrid-columnHeaderTitle"
                        data-mui-internal-clone-element="true"
                      >
                        Cash Balance
                      </div>
                    </div>
                    <div
                      class="MuiDataGrid-iconButtonContainer css-ltf0zy-MuiDataGrid-iconButtonContainer"
                    >
                      <button
                        aria-label="Sort"
                        class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeSmall css-1pe4mpk-MuiButtonBase-root-MuiIconButton-root"
                        tabindex="-1"
                        title="Sort"
                        type="button"
                      >
                        <svg
                          aria-hidden="true"
                          class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall MuiDataGrid-sortIcon css-ptiqhd-MuiSvgIcon-root"
                          data-testid="ArrowUpwardIcon"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M4 12l1.41 1.41L11 7.83V20h2V7.83l5.58 5.59L20 12l-8-8-8 8z"
                          />
                        </svg>
                        <span
                          class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                        />
                      </button>
                    </div>
                  </div>
                  <div
                    class="MuiDataGrid-menuIcon"
                  >
                    <button
                      aria-expanded="false"
                      aria-haspopup="menu"
                      aria-label="Menu"
                      class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeSmall MuiDataGrid-menuIconButton css-1pe4mpk-MuiButtonBase-root-MuiIconButton-root"
                      data-mui-internal-clone-element="true"
                      id=":rp:"
                      tabindex="-1"
                      type="button"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall css-ptiqhd-MuiSvgIcon-root"
                        data-testid="TripleDotsVerticalIcon"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M12 8c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2zm0 2c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm0 6c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z"
                        />
                      </svg>
                      <span
                        class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                      />
                    </button>
                  </div>
                </div>
                <div
                  class="MuiDataGrid-columnSeparator MuiDataGrid-columnSeparator--sideRight"
                  style="min-height: 56px; opacity: 1;"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiDataGrid-iconSeparator css-i4bv87-MuiSvgIcon-root"
                    data-testid="SeparatorIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M11 19V5h2v14z"
                    />
                  </svg>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="MuiDataGrid-virtualScroller css-axafay-MuiDataGrid-virtualScroller"
          style="overflow-x: hidden;"
        >
          <div
            class="MuiDataGrid-overlayWrapper css-ql19fo-MuiDataGrid-overlayWrapper"
          >
            <div
              class="MuiDataGrid-overlayWrapperInner css-1akuw9y-MuiDataGrid-overlayWrapperInner"
              style="height: -56px; width: 0px;"
            >
              <span
                class="MuiLinearProgress-root MuiLinearProgress-colorPrimary MuiLinearProgress-indeterminate css-eglki6-MuiLinearProgress-root"
                role="progressbar"
              >
                <span
                  class="MuiLinearProgress-bar MuiLinearProgress-barColorPrimary MuiLinearProgress-bar1Indeterminate css-5ir5xx-MuiLinearProgress-bar1"
                />
                <span
                  class="MuiLinearProgress-bar MuiLinearProgress-barColorPrimary MuiLinearProgress-bar2Indeterminate css-1r8wrcl-MuiLinearProgress-bar2"
                />
              </span>
            </div>
          </div>
          <div
            class="MuiDataGrid-virtualScrollerContent MuiDataGrid-virtualScrollerContent--overflowed css-1kwdphh-MuiDataGrid-virtualScrollerContent"
            style="width: auto; height: 52px; min-height: auto;"
          >
            <div
              class="MuiDataGrid-virtualScrollerRenderZone css-s1v7zr-MuiDataGrid-virtualScrollerRenderZone"
              style="transform: translate3d(0px, 0px, 0px);"
            >
              <div
                aria-rowindex="2"
                aria-selected="false"
                class="MuiDataGrid-row MuiDataGrid-row--editable MuiDataGrid-row--lastVisible"
                data-id="user-id"
                data-rowindex="0"
                role="row"
                style="max-height: 52px; min-height: 52px;"
              >
                <div
                  aria-colindex="1"
                  aria-colspan="1"
                  class="MuiDataGrid-cell MuiDataGrid-cell--textLeft MuiDataGrid-withBorderColor"
                  data-colindex="0"
                  data-field="id"
                  role="cell"
                  style="min-width: 0; max-width: 0; min-height: 52px; max-height: 52px;"
                  tabindex="-1"
                >
                  <div
                    class="MuiDataGrid-cellContent"
                    title="user-id"
                  >
                    user-id
                  </div>
                </div>
                <div
                  aria-colindex="2"
                  aria-colspan="1"
                  class="MuiDataGrid-cell MuiDataGrid-cell--textLeft MuiDataGrid-withBorderColor"
                  data-colindex="1"
                  data-field="username"
                  role="cell"
                  style="min-width: 0; max-width: 0; min-height: 52px; max-height: 52px;"
                  tabindex="-1"
                >
                  <div
                    class="MuiDataGrid-cellContent"
                    title="jdoe"
                  >
                    jdoe
                  </div>
                </div>
                <div
                  aria-colindex="3"
                  aria-colspan="1"
                  class="MuiDataGrid-cell MuiDataGrid-cell--textRight MuiDataGrid-cell--editable MuiDataGrid-withBorderColor"
                  data-colindex="2"
                  data-field="cashBalance"
                  role="cell"
                  style="min-width: 0; max-width: 0; min-height: 52px; max-height: 52px;"
                  tabindex="-1"
                >
                  <div
                    class="MuiDataGrid-cellContent"
                    title="100"
                  >
                    100
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiDataGrid-footerContainer MuiDataGrid-withBorderColor css-wop1k0-MuiDataGrid-footerContainer"
      >
        <div />
        <div
          class="MuiTablePagination-root css-rtrcn9-MuiTablePagination-root"
        >
          <div
            class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular MuiTablePagination-toolbar css-78c6dr-MuiToolbar-root-MuiTablePagination-toolbar"
          >
            <div
              class="MuiTablePagination-spacer css-1psng7p-MuiTablePagination-spacer"
            />
            <p
              class="MuiTablePagination-selectLabel css-pdct74-MuiTablePagination-selectLabel"
              id=":rb:"
            >
              Rows per page:
            </p>
            <div
              class="MuiInputBase-root MuiInputBase-colorPrimary css-16c50h-MuiInputBase-root-MuiTablePagination-select"
            >
              <div
                aria-expanded="false"
                aria-haspopup="listbox"
                aria-labelledby=":rb: :ra:"
                class="MuiSelect-select MuiTablePagination-select MuiSelect-standard MuiInputBase-input css-194a1fa-MuiSelect-select-MuiInputBase-input"
                id=":ra:"
                role="button"
                tabindex="0"
              >
                100
              </div>
              <input
                aria-hidden="true"
                aria-invalid="false"
                class="MuiSelect-nativeInput css-yf8vq0-MuiSelect-nativeInput"
                tabindex="-1"
                value="100"
              />
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiSelect-icon MuiTablePagination-selectIcon MuiSelect-iconStandard css-pqjvzy-MuiSvgIcon-root-MuiSelect-icon"
                data-testid="ArrowDropDownIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M7 10l5 5 5-5z"
                />
              </svg>
            </div>
            <p
              class="MuiTablePagination-displayedRows css-levciy-MuiTablePagination-displayedRows"
            >
              1–1 of 1
            </p>
            <div
              class="MuiTablePagination-actions"
            >
              <button
                aria-label="Go to previous page"
                class="MuiButtonBase-root Mui-disabled MuiIconButton-root Mui-disabled MuiIconButton-colorInherit MuiIconButton-sizeMedium css-zylse7-MuiButtonBase-root-MuiIconButton-root"
                disabled=""
                tabindex="-1"
                title="Go to previous page"
                type="button"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                  data-testid="KeyboardArrowLeftIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M15.41 16.09l-4.58-4.59 4.58-4.59L14 5.5l-6 6 6 6z"
                  />
                </svg>
              </button>
              <button
                aria-label="Go to next page"
                class="MuiButtonBase-root Mui-disabled MuiIconButton-root Mui-disabled MuiIconButton-colorInherit MuiIconButton-sizeMedium css-zylse7-MuiButtonBase-root-MuiIconButton-root"
                disabled=""
                tabindex="-1"
                title="Go to next page"
                type="button"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                  data-testid="KeyboardArrowRightIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M8.59 16.34l4.58-4.59-4.58-4.59L10 5.75l6 6-6 6z"
                  />
                </svg>
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
